spec/models/project_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Project, type: :model do
+  subject {
+    Project.new(
+      repo_url: 'https://github.com/learner/repo',
+      user_id: user.id,
+      lesson_id: lesson.id
+    )
+  }
+  let(:user) { double('User', id: 1) }
+  let(:lesson) { double('Lesson', id: 1) }
+
+  it { is_expected.to belong_to(:user) }
+  it { is_expected.to belong_to(:lesson) }
+
+  it do
+    is_expected.to validate_presence_of(:repo_url)
+      .with_message('must start with http://github.com/ or https://github.com/')
+  end
+
+  it do
+    is_expected.to allow_values(
```

Comment:
Can we replace these two tests with a shoulda matcher please?
the shoulda matcher `allow_value` will work well here